test-data/unit/python2eval.test

```diff
@@ -396,11 +403,12 @@ def f(x: unicode) -> int: pass
 def f(x: bytearray) -> int: pass
 [out]
 _program.py:2: error: No overload variant of "f" matches argument types [builtins.int]
+_program.py:5: error: No overload variant of "f" matches argument types [builtins.bytes]
```

Comment:
Because there's no overload for bytes, and there's no equivalency between bytes and bytearray. Not sure I like this.